README.md

```diff
@@ -39,21 +39,41 @@ These modules are designed to be flexible and allow for reordering with few exce
 
 NeMo Curator currently requires Python 3.10 and the GPU accelerated modules require CUDA 12 or above installed in order to be used.
 
+### PyPi
+
 NeMo Curator can be installed manually by cloning the repository and installing as follows -
 
 For CPU only modules:
 ```
-pip install .
+pip install nemo-curator
 ```
 
-For CPU + CUDA accelerated modules
+For CPU + CUDA accelerated modules:
 ```
-pip install --extra-index-url https://pypi.nvidia.com ".[cuda12x]"
+pip install --extra-index-url https://pypi.nvidia.com nemo-curator[cuda12x]
 ```
 
 ### NeMo Framework Container
 
-NeMo Curator is available in the [NeMo Framework Container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo). The NeMo Framework Container provides an end-to-end platform for development of custom generative AI models anywhere. The latest release of NeMo Curator comes preinstalled in the container.
+The latest release of NeMo Curator is preinstalled in the [NeMo Framework Container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo). The NeMo Framework Container provides an end-to-end platform for development of custom generative AI models anywhere. If you want the latest commit inside the container, uninstall the existing version using
+```
+pip uninstall nemo-curator
```

Comment:
I'm not sure if this is needed or not. But no objection to keeping it in clarity.